rancher2/data_source_rancher2_project.go

```diff
@@ -0,0 +1,90 @@
+package rancher2
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/rancher/norman/types"
+)
+
+func dataSourceRancher2Project() *schema.Resource {
+	return &schema.Resource{
+		Exists: dataSourceRancher2ProjectExists,
+		Read:   dataSourceRancher2ProjectRead,
+
+		Schema: map[string]*schema.Schema{
+			"uuid": {
```

Comment:
Just a format question here, first `required`, next `optional` and/or `computed` arguments. Could you please move "uuid" argument to the end of the list??